label_studio/tasks/migrations/0043_auto_20230825.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.19 on 2023-08-10 23:04
+
+from django.db import migrations
+from tasks.functions import fill_predictions_project
+
+def forward(apps, schema_editor):
+    fill_predictions_project()
+
+
+def backwards(apps, schema_editor):
+    pass
+
+
+class Migration(migrations.Migration):
```

Comment:
Figure the easiest thing to do is to just create a duplicate data migration that updates any new Predictions with no project_id linkage.